### DISCUSSION TITLE
All hail folke, creator of Lazy, writer of docs!

### DISCUSSION THREAD

**Discussion by tommyalatalo** [LazyVim/LazyVim#425] [General]
I just had to drop a note here today after I accidentally found [lazyvim.org](https://www.lazyvim.org/) while fixing something in my lazy config.

@folke, not only did you write one of the best extensions to neovim, but you also had the decency to document it like a boss! My hat's off to you, [lazyvim.org](https://www.lazyvim.org/) is a great reference and I'll be pointing myself, friends and coworkers over there for sure.

This discovery really made my day! :D

**Comment by folke** [comment]
Thanks! Happy to know there are actual people out there that **do** read the docs ;)
